web/src/engine/osk/src/keyboard-layout/oskLayer.ts

```diff
@@ -76,6 +81,21 @@ export default class OSKLayer {
     this.capsKey     = this.findKey('K_CAPS');
     this.numKey      = this.findKey('K_NUMLOCK');
     this.scrollKey   = this.findKey('K_SCROLL');
+
+    if(this.spaceBarKey) {
+      const spacebarLabel = this.spaceBarKey.label;
+      let tButton = this.spaceBarKey.btn;
+
+      if (typeof (tButton.className) == 'undefined' || tButton.className == '') {
+        tButton.className = 'kmw-spacebar';
+      } else if (tButton.className.indexOf('kmw-spacebar') == -1) {
+        tButton.className += ' kmw-spacebar';
+      }
```

Comment:
```suggestion
      tButton.classList.add('kmw-spacebar');
```

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList